python/GafferSceneUI/InstancerUI.py

```diff
@@ -95,35 +99,106 @@
 			"description",
 			"""
 			The scene containing the prototypes to be applied to
-			each vertex. Specify multiple prototypes by parenting
-			them at the root of the scene :
-
-			- /prototype0
-			- /prototype1
-			- /prototype2
+			each vertex. Use the `prototypeMode` and associated
+			plugs to control the mapping between prototypes and
+			instances.
 
-			Note that the prototypes are not limited to being a
-			single object : they can each have arbitrary child
-			hierarchies.
+			Note that the prototypes are not limited to being a single
+			object -- they can have arbitrary child hierarchies.
```

Comment:
Oh I've no idea what's proper grammatically... I guess I use : for headings only, ; when I'm writing prose and -- when I want to draw someone's attention mid sentence. Here I probably should have replaced the existing : with a ; but I chose -- to be more attention grabbing. I'd 100% defer to @medubelko on what it should be though.

Reply:
A dash (or hyphen, since there are still places were UTF-8 is dangerous) is fine here. This is a reasonable use of it to grab the reader's attention. The rules for typographic conundrums vary widely, and often come down to the editor's personal preference in the style guide. I haven't published a rule for this to give everyone some leeway, but since we haven't been using double hyphens systematically, let's keep it a single hyphen in place of an en dash.

Reply:
Addressed in f0a3f3fe1043282a94eb5fe978748c705ba5cbc6